src/coreclr/jit/block.h

```diff
@@ -629,6 +618,13 @@ struct BasicBlock : private LIR::Range
         }
     }
 
+    // Set block weight to zero, and set run rarely flag.
+    //
+    void bbSetRunRarely()
+    {
+        this->scaleBBWeight(BB_ZERO_WEIGHT);
```

Comment:
As far as I understand, previous `bbSetRunRarely` used to do nothing if the block has a profile weight

Reply:
Right. If you call this you are going to override PGO.